woocommerce/woocommerce
# Replaced deprecated Jetpack::is_staging_site call

Replaces #25665 .

### All Submissions:






### Changes proposed in this Pull Request:



Closes #25664 .

### How to test the changes in this Pull Request:

1. Get a new JN instance
2. Build this package and upload it as a plugin to JN
3. Go through the OBW and 
- allow tracking, 
- activate Jetpack, 
4. you shouldn't see any error (ideally). Or at least not the error saying Notice: Jetpack::is_staging_site is deprecated since version jetpack-8.1! Use /Automattic/Jetpack/Status->is_staging_site instead.
4. Get another JN without Jetpack
5. Install Jetpack older than 8.1, e.g. 7.9 or 8.0
6. Verify that there are no error during OBW when activating Jetpack.

### Other information:




### Changelog entry

> Fix - Replaced deprecated Jetpack::is_staging_site call.
